packages/core/src/new-api/types/module-builder.ts

```diff
@@ -211,6 +225,12 @@ export interface IgnitionModuleBuilder {
     options?: ReadEventArgumentOptions
   ): ReadEventArgumentFuture;
 
+  send(
+    to: string | AddressResolvableFuture,
+    data: string,
+    options?: SendDataOptions
+  ): SendDataFuture;
```

Comment:
@zoeyTM what if we do
```suggestion
  send(
    id: string,
    to: string | AddressResolvableFuture,
    value?: bigint,
    data?: string,
    options?: SendDataOptions
  ): SendDataFuture;
```

and only leave `after` and `from` in the options?

My reasoning:

1) required `id`, as it's unclear how to infer an useful one.
2) `value` and `data` are both "first-class" arguments, as they are both intrinsic to the operation the user is asking to do, while options are more metadata-like.
3) `value` and `data` are both optional because you may want to use one and/or the other (e.g. send ETH, send data to a non-payable contract)
4) `value` before `data` because it's simpler to omit with a `0` if you only want to send data.

Reply:
> required `id`, as it's unclear how to infer an useful one.

definitely agree with this one

> `value` and `data` are both "first-class" arguments, as they are both intrinsic to the operation the user is asking to do, while options are more metadata-like.

i agree and see your point, but i'm torn because i also like the consistency of keeping `value` as an option like it is everywhere else. Ultimately I think I agree with making it a first class param for this function though

Reply:
My other thought was actually moving `data` into the options object for this function as well, to better facilitate that any of those fields were optional